lib/chai/core/assertions.js

```diff
@@ -2030,43 +2968,129 @@ module.exports = function (chai, _) {
 
 
   /**
-   * ### .change(function)
+   * ### .change(subject[, prop[, msg]])
+   *
+   * When invoked as a method with one argument, `.change` asserts that the
+   * given function `subject` returns a different value when it's invoked before
+   * the target function compared to when it's invoked afterward. Strict (`===`)
+   * equality is used to compare `subject`'s before and after return values.
+   * `.change` should typically only be used when it has been negated by adding
```

Comment:
🛑 should be something like "`.change` - when used with one argument - should typically only be used with `.not`."